numpy/core/src/umath/scalarmath.c.src

```diff
@@ -167,7 +167,7 @@ static NPY_INLINE int
     }
 #if @neg@
     else if (b == -1 && a == NPY_MIN_@NAME@) {
-        *out = a / b;
+        *out = NPY_MIN_@NAME@;
         return NPY_FPE_OVERFLOW;
```

Comment:
This is suspicious: is this line really not hit during tests?

Reply:
Yep https://github.com/numpy/numpy/pull/21507 has tests that covers this.

Reply:
I think both PRs have tests for this, so suspect that it is just in-lining/optimization leading to coverage errors (that might be the first time I have seen coverage failing, but...).

@ganesh-k13 if you prefer it to not be included here, maybe just push the change yourself?  (We have to mark that one additional test as `skip` then probably.)  Otherwise, I am happy to merge as-is.

Yes, it would be nice to follow-up to expand the tests so that they raise the error, and there is still lots of little stuff to clear out in `multiplication`... But at this point it feels like iterating here is getting tricky.